mix.exs

```diff
@@ -1,19 +1,19 @@
 defmodule Cluster.Mixfile do
   use Mix.Project
 
-  @version "3.4.1"
+  @version "4.0.0"
   @source_url "https://github.com/bitwalker/libcluster"
 
   def project do
     [
-      app: :libcluster,
+      app: :fresha_libcluster,
```

Comment:
please make sure that this (and all the references to this name) match the repository name.